ruby/Ch03-Msgy-Schema/Msgy.rb

```diff
@@ -147,7 +150,7 @@ def generate_key(owner, type, date)
 end
 
 # Setup our repositories
-client = Riak::Client.new(protocol: 'pbc', pb_port: 10017)
+client = Riak::Client.new(pb_port: 17017)
```

Comment:
The `pb_port: 17017` is going to break on regular Mac OS Riak installs, gimme a sec to fix it…